modules/app/src/main/resources/admin/tools/main/main.html

```diff
@@ -33,7 +33,8 @@
     <link rel="stylesheet" type="text/css" href="{{assetsUri}}/admin/common/styles/lib.css">
     <link rel="stylesheet" type="text/css" href="{{assetsUri}}/styles/contentlib.css">
     <script type="text/javascript" src="{{assetsUri}}/admin/common/js/lib.js"></script>
-    <script type="text/javascript" src="{{assetsUri}}/lib/vendors.js"></script>
+    <script type="text/javascript" src="{{assetsUri}}/lib/ckepath.js" data-asset-url="{{assetsUri}}"></script>
+    <script type="text/javascript" src="{{assetsUri}}/lib/ckeditor/ckeditor.js"></script>
```

Comment:
Can't we import './ckeditor/ckeditor' inside `ckepath.js`, let Webpack bundle them together and avoid having to add two references to cke assets in all of our html templates?

Reply:
The idea in general is that CKEDITOR_BASEPATH to be set before invoking ckeditor.js that requires this var to be already set,  however we can't put import of ckeditor.js on the first line in the ckepath.js - that will be the same problem when this required var won't be set when ckeditor.js will be executed. So we need to keep it in a separate modules and make ckepath.js always come before ckeditor.js to guarantee CKEDITOR_BASEPATH is set